ui/app/css/itcss/components/simple-dropdown.scss

```diff
@@ -26,14 +26,13 @@
 .simple-dropdown__options {
```

Comment:
Nit: These styles are now unused, so they can be removed.

Note that I did see a few cases of `simpledropdown` styles being used _outside_ of the `SimpleDropdown` component. `send-asset-row` uses `simple-dropodown__caret` and `dropdown-search-list` uses `simple-dropdown__close-area`. Those two should probably be migrated into the styles for those components.

Reply:
Good finds!  I've updated those external components and removed the unnecessary CSS!